src/TrackPanelAx.cpp

```diff
@@ -368,33 +363,8 @@ wxAccStatus TrackPanelAx::GetName( int childId, wxString* name )
          }
          else
          {
-            *name = t->GetName();
-            if( *name == t->GetDefaultName() )
-            {
-               /* i18n-hint: The %d is replaced by the number of the track.*/
-               name->Printf(_("Track %d"), TrackNum( t ) );
-            }
-
-            t->TypeSwitch(
-               [&](const LabelTrack *) {
-                  /* i18n-hint: This is for screen reader software and indicates that
-                     this is a Label track.*/
-                  name->Append( wxT(" ") + wxString(_("Label Track")));
-               },
-               [&](const TimeTrack *) {
-                  /* i18n-hint: This is for screen reader software and indicates that
-                     this is a Time track.*/
-                  name->Append( wxT(" ") + wxString(_("Time Track")));
-               }
-#ifdef USE_MIDI
-                ,
-               [&](const NoteTrack *) {
-                  /* i18n-hint: This is for screen reader software and indicates that
-                     this is a Note track.*/
-                  name->Append( wxT(" ") + wxString(_("Note Track")));
-               }
-#endif
-            );
+            name->Printf(_("Track %d"), TrackNum(t));
```

Comment:
Do not delete the i18n_hint comment that helps the translators

Reply:
I've found multiple instances, of the same string in different places and with different comments (or without them), and didn't find an approriate one for this case, but missed that one: `/* i18n-hint: The %d is replaced by the number of the track.*/`
Thanks.

Reply:
If a translatable string literal is repeated exactly in more than one place, and without a context string, then it is enough the one occurrencde has a comment.